.github/actions/trivy/action.yaml

```diff
@@ -3,16 +3,18 @@ description: 'action that runs trivy '
 runs:
   using: "composite"
   steps:
-    - name: Install trivy
+    - name: Setup reports folder
       run: |
-        apk update
-        apk add curl
-        curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sh -s -- -b /usr/local/bin
         mkdir trivy-reports
       shell: sh
-    - name: Scan Image
-      run: trivy image --input ${GITHUB_SHA}_image.tar
-              -o trivy-reports/image.txt
-              --exit-code 1
-              --severity="UNKNOWN,MEDIUM,HIGH,CRITICAL"
-      shell: sh
+    - name: Run Trivy on deployment.yaml
```

Comment:
`on deployment.yaml`? doesn't this scan the whole image?

Reply:
copy-paste :sweat_smile: